src/SockPerf.cpp

```diff
@@ -1564,11 +1568,11 @@ static int proc_mode_server( int id, int argc, const char **argv )
 		printf("\n");
 		printf("Usage: " MODULE_NAME " %s [options] [args]...\n", sockperf_modes[id].name);
 		printf(" " MODULE_NAME " %s\n", sockperf_modes[id].name);
-		printf(" " MODULE_NAME " %s [-i ip] [-p port] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);
+		printf(" " MODULE_NAME " %s [-i ip] [-p port] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);
 #ifndef WIN32
-		printf(" " MODULE_NAME " %s -f file [-F s/p/e] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);
+		printf(" " MODULE_NAME " %s -f file [-F s/p/e] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);
 #else
-		printf(" " MODULE_NAME " %s -f file [-F s] [--rx-mc-if ip] [--tx-mc-if ip]\n", sockperf_modes[id].name);
+		printf(" " MODULE_NAME " %s -f file [-F s] [--rx-mc-if ip] [--tx-mc-if ip] [--mc-source-ip]\n", sockperf_modes[id].name);
 #endif
```

Comment:
wouldn't it be better to have the new name **[--mc-source-ip]**
in the format of the old names **[--rx-mc-if ip] [--tx-mc-if ip]**